enterprise/server/cmd/ci_runner/main.go

```diff
@@ -2444,6 +2515,23 @@ func runCredentialHelper() error {
 	}
 }
 
+func runBazelWrapper() error {
+	rootPath := os.Getenv("CI_RUNNER_ROOT")
+	bazelCmd := os.Getenv("BAZEL_BIN")
```

Comment:
nit: unset these env vars after reading them and before calling bazel, just to avoid people depending on these unnecessarily